apiary.apib

```diff
@@ -1556,41 +3369,1468 @@ Fetches a bundle metadata by identifier
+                + validUnitil (number) - the unix timestamp of the time when invite become invalid
+                + createdBy (string) - the account address of the invite creator
+                + createdOn (number) - the unix timestamp of the invite creation time
+        + meta (object) - the metadata
+            + code (number) - the status code
+            + count (number) - the count of the results
+        + pagination (object) - the helper object to handle pagination
+            + hasNext (boolean) - tells if there is a next page
+            + hasPrevious (boolean) - tells if there is a previous page
+
+    + Body
+
+        {
+            "data": [
+                {
+                    "organizationId": 9,
+                    "email": "test@example.com",
+                    "subject": "Invitation",
+                    "inviteId": "10",
+                    "invitationLink": "inivite.example.com".
+                    "sent": true,
+                    "validUntil": 1633594494,
+                    "createdBy": "0x0C6C7264E509430bcFa2469bA4eB738c94e6C75A",
+                    "createdOn": 1633594494
+                }
+            ],
+            "meta": {
+                "code": 200,
+                "count": 1
+            },
+            "pagination": {
+                "hasNext": false,
+                "hasPrevious": false
+            }
+        }
+
+
+## /organization/invite/{inviteId}/exists
+#### Check if invite exists [GET]
++ Parameters
+
+    + inviteId (number, required) - the invitaion Id
++ Response 200 (application/json)
+
+    + Attributes
+
+        + data (boolean) - shows if invite exists
+        + meta (object) - the metadata
+            + code (string) - the status code
+
+    + Body
+
+            {
+                "data": true,
+                "meta": {
+                    "code": 200
+                }
+            }
+
+## /organization/invite/{inviteId}/accept
+#### Accept the invitation [POST]
+* Parameters
+
+    + inviteId (number, required) - the invitation Id
+
++ Request
+
+    + Attributes
+
+        + address (string) - the account of the one who accepts invitation
+
+    + Body
+
+            {
+                "address": "0x203C394e59915f85B9c222f6E15ce1FFF699559E"
+            }
+
++ Response 200 (application/json)
+
+    Accepted.
+
+    + Body
+
++ Response 404 (application/json)
+
+    Invite doesn't exist or the metadata for it is missing
+
+    + Body
+
+## /organization/invite/{inviteId}
+#### Delete the invitation [DELETE]
++ Parameters
+
+    + inviteId (number, required) - the invitation Id
+
++ Response 200 (application/json)
+
+    Deleted.
+
+    + Body
+
++ Response 404 (application/json)
+
+    Invite doesn't exist or the metadata for it is missing
+
+    + Body
+
+## /organization/invite/resend
+__Warning__ super_account permission is required
+#### Resend all invitation [GET]
++ Response 200 (application/json)
+
+    + Body
+
+
+## /organization/invite/
```

Comment:
Same as /organization/invite{?perPage,page,createdBy,data}  ?

Reply:
There is GET and POST calls description. I'll try to make it unambiguous